packages/user-group-service/src/users/typedef.graphql

```diff
@@ -97,6 +97,55 @@ input UserInput {
   updatedOn: DateTime
 }
 
+enum fieldType {
+  uid
+  cn
+  rhatUuid
```

Comment:
```suggestion
  rhatUUID
```

Reply:
@riginoommen @deshmukhmayur `rhatUuid` passed as a query param to the URL, though it is not case sensitive should we make it consistent with what(LDAP queries) is going to be removed later on?

Reply:
For the consitency to One Platform on the variable names, Made the variable name to `rhatUUID`.